amr-wind/wind_energy/ABLBoundaryPlane.cpp

```diff
@@ -144,21 +144,32 @@ void InletData::read_data_native(
     const int lev,
     const Field* fld,
     const amrex::Real time,
-    const amrex::Vector<amrex::Real>& times)
+    const amrex::Vector<amrex::Real>& times,
+    const bool erf_multiblock)
 {
     const size_t nc = fld->num_comp();
     const int nstart =
         static_cast<int>(m_components[static_cast<int>(fld->id())]);
 
-    const int idx = utils::closest_index(times, time);
+    int idx;
```

Comment:
1.  Is the intention here to have `times[idx] < time <= times[idx+1]` for multiblock instead of `times[idx] <= time < times[idx+1]`? 
2. I'm a bit confused but the `1+eps` checks below, are they necessary with `lbound`?